### ISSUE TITLE
Gpu problem(Blas SGEMM launch failed )

### ISSUE THREAD

**Issue by iamheidi** [Qidian213/deep_sort_yolov3#80]
Hi.There.
I succeeded in the CPU. But gpu is a problem.

like as:
> Caused by op 'conv2d_3/convolution', defined at:
>   File "demo_heidi.py", line 117, in <module>
>     main(YOLO())
>   File "/home/heidi/GitProject/tracking/deep_sort_yolov3/yolo.py", line 80, in __init__
>     self.boxes, self.scores, self.classes = self.generate()
>   File "/home/heidi/GitProject/tracking/deep_sort_yolov3/yolo.py", line 101, in generate
>     self.yolo_model = load_model(model_path, compile=False)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/engine/saving.py", line 419, in load_model
>     model = _deserialize_model(f, custom_objects, compile)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/engine/saving.py", line 225, in _deserialize_model
>     model = model_from_config(model_config, custom_objects=custom_objects)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/engine/saving.py", line 458, in model_from_config
>     return deserialize(config, custom_objects=custom_objects)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/layers/__init__.py", line 55, in deserialize
>     printable_module_name='layer')
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/utils/generic_utils.py", line 145, in deserialize_keras_object
>     list(custom_objects.items())))
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/engine/network.py", line 1032, in from_config
>     process_node(layer, node_data)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/engine/network.py", line 991, in process_node
>     layer(unpack_singleton(input_tensors), **kwargs)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/engine/base_layer.py", line 457, in __call__
>     output = self.call(inputs, **kwargs)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/layers/convolutional.py", line 171, in call
>     dilation_rate=self.dilation_rate)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 3650, in conv2d
>     data_format=tf_data_format)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/tensorflow/python/ops/nn_ops.py", line 780, in convolution
>     return op(input, filter)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/tensorflow/python/ops/nn_ops.py", line 868, in __call__
>     return self.conv_op(inp, filter)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/tensorflow/python/ops/nn_ops.py", line 520, in __call__
>     return self.call(inp, filter)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/tensorflow/python/ops/nn_ops.py", line 204, in __call__
>     name=self.name)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/tensorflow/python/ops/gen_nn_ops.py", line 957, in conv2d
>     data_format=data_format, dilations=dilations, name=name)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
>     op_def=op_def)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py", line 488, in new_func
>     return func(*args, **kwargs)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3274, in create_op
>     op_def=op_def)
>   File "/home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1770, in __init__
>     self._traceback = tf_stack.extract_stack()
> 
> InternalError (see above for traceback): Blas SGEMM launch failed : m=43264, n=32, k=64
> 	 [[node conv2d_3/convolution (defined at /home/heidi/.pyenv/versions/pyenv361/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3650)  = Conv2D[T=DT_FLOAT, _class=["loc:@batch_normalization_3/cond/FusedBatchNorm/Switch"], data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](leaky_re_lu_2/LeakyRelu, conv2d_3/kernel/read)]]
> 	 [[{{node boolean_mask_29/GatherV2/_591}} = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_6054_boolean_mask_29/GatherV2", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]
> 

So I tried add code global_variables_initializer and 

```
gpu_options = tf.GPUOptions(per_process_gpu_memory_fraction=0.333)
self.sess = tf.Session(config=tf.ConfigProto(gpu_options=gpu_options))
```

in yolo.py

But still an error occurs.

**Comment by iamheidi** [comment]
I solved. I did not install the 4 cuda9.0 patches. After installing the 4 patches, the code was run normally.
